Source/Core/Core/NetPlayServer.cpp

```diff
@@ -1269,7 +1269,23 @@ bool NetPlayServer::StartGame()
 
   // no change, just update with clients
   if (!m_host_input_authority)
+  {
+    if (Config::Get(Config::NETPLAY_AUTO_BUFFER))
+    {
+      unsigned int latency = 0;
+      for (const auto& player_entry : m_players)
+      {
+        latency += player_entry.second.ping;
+      }
+
+      const unsigned int pad_buffer =
+          ((latency * (static_cast<int>(m_players.size()) - 1)) / 15) + 1;
```

Comment:
Maybe this calculation should be placed into a reusable function, since it's used once here in core code, and also used in the frontend code.

Note that this code also results in a signed/unsigned implicit conversion, since player size is converted to an int and then that side of the expression. Maybe make the signedness conversion explicit (or avoidable if possible)

Reply:
This is a good point, and per the next comment, I forgot to use u32. I will look at a cleaner implementation.